packages/commonjs-shim/src/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const CJSyntaxRe = /__filename|__dirname|require\(|require\.resolve\(/;
+
+export const CJSShim = `
+// -- CommonJS Shims --
+import cjsUrl from 'url';
+import cjsPath from 'path';
+import cjsMod from 'module';
```

Comment:
```suggestion
import cjsModule from 'module';
```

let's be verbose. it doesn't cost us anything and will make debugging easier.

Reply:
done.